### ISSUE TITLE
Anomaly Enhancement

### ISSUE THREAD

**Issue by captaincobbs** [wakystuf/ESG-Mod#872]
Completely disregard this document :
https://docs.google.com/document/d/1GsgUxXha4-4wWr_M5FiglITY5gUgCx-4-Q2X2qJ7TQU/edit

Actual Enhanced Anomalies Effects :
https://docs.google.com/document/d/1cYq_owRRVrCUuwQJqouTZdtsCRJ7uHQUzBQIvCgDEtE/edit

- [x] #1184
- [x] #1192
- [x] Work out how to make anomaly enhancement Ecologist-only
- [x] #1193
- [x] #1194
- [x] Reconfigure the tech tree to put anomaly enhancement techs at the right point
- [x] Balance pass
- [x] Documentation

Merge Advanced Non-Baryonics into Ontological Rev-Eng, Move Anomaly Reduction to Climate Engineering
Move Modded Terraformation to new technologies
Give the neutral / negative anomaly traits a "x anomaly is spawned on your home planet" message, only gives anomaly stats

**Comment by Arpharnator** [comment]
Gamer list

**Comment by captaincobbs** [comment]
The more you speak the less intelligent I'm convinced you are

**Comment by Arpharnator** [comment]
mirror

**Comment by Arpharnator** [comment]
-1 IQ

**Comment by Arpharnator** [comment]
-1 IQ

**Comment by Arpharnator** [comment]
-1 IQ

**Comment by Arpharnator** [comment]
-1 IQ

**Comment by Arpharnator** [comment]
-1 IQ

**Comment by wakystuf** [comment]
My work here is done:

```xml
<AnomalyReductionDefinition Name="AnomalyReduction16" Category="CategoryAnomalyReduction">
        <CustomCost ResourceName="SystemProduction">480 * (1 - Property(StockLocation, @ClassColonizedStarSystem, BuildingProductionCostReduction)) * (1 - Property(StockLocation, @ClassColonizedStarSystem, GameSpeedProductionCostReduction))</CustomCost>
        <TechnologyPrerequisite UnlockHidden="true" Flags="Technology,Prerequisite,Discard">TechnologyAnomalyReduction1</TechnologyPrerequisite>
        <BaseAnomaly Name="PlanetAnomaly16"/>
        <ResultAnomaly Name="PlanetAnomaly01"/>
    </AnomalyReductionDefinition>
```

**Comment by FluffiestPrincess** [comment]
This is a pretty big one; it might be best to split the work over multiple issues and assign to different people, so I made it into a project column. I've also created a new branch for it, just as a clone of 1-5-1 so far (Cobbs asked me to).

**Comment by FluffiestPrincess** [comment]
_Components have been placed in first post, so we get a neat little progress bar._

**Comment by captaincobbs** [comment]
Adding them to the tech tree is easy, I'll try to get to that tonight. Doing it early also means we don't need to go through all the anomaly enhancement constructibles and add the tech prerequisite afterwards. I can also make a prerequisite that makes it ecologists only.

**Comment by FluffiestPrincess** [comment]
A few requests:

- [ ] Can you think of something other than "+Hyperium" for Shattered Crust? Hyperium is apparently an exotic form of hydrogen and shouldn't really be in the planet's crust.
- [ ] Please work out whether the "all tags" effect for Garden of Eden actually works, and if not, tell me what you actually want
- [ ] Please decide on what effect you want for Dust Ruins

**Comment by Provolonage** [comment]
oh wait is this actually happening, I think cobbs and I need to take another look at this because so much changed since we did the initial write up.

**Comment by FluffiestPrincess** [comment]
Yes, please, and the sooner the better. I've already written almost all of the flavour text, but I'm ok to change it if I can do it soon.

**Comment by Provolonage** [comment]
oh don't worry it would be mostly numbers tweaks, i think some of the proposed ideas would be too good or too weak, I don't think any of the anoms would change in effect significantly.

**Comment by FluffiestPrincess** [comment]
Ok, but still:

- Shattered Crust shouldn't give Hyperium; that doesn't make sense with the in-game fiction.
- If the Garden of Eden "has all planet tags" doesn't work, I need to know what the alternate effect is.
- Lemme know what "Dust Ruins" does when enhanced; that was left out of the document by mistake.

**Comment by captaincobbs** [comment]
> Ok, but still:
> 
> * Shattered Crust shouldn't give Hyperium; that doesn't make sense with the in-game fiction.
> * If the Garden of Eden "has all planet tags" doesn't work, I need to know what the alternate effect is.
> * Lemme know what "Dust Ruins" does when enhanced; that was left out of the document by mistake.

Shattered Crust could give Titanium I suppose
Garden of Eden I'm thinking No disapproval due to planet type (force happiness raw to 0), +2 FIDSI per Population
Dust Ruins would be cool if it gave +20 Dust per System Level and an additional +1 dust per pop when enhanced

**Comment by wakystuf** [comment]
@Arpharnator can you write some simple documentation so that @Kil10naire can put it on the changelog? The discussion above is a) out-of-date/inaccurate and b) hard to follow. For the details of each anomaly, feel free to link the final version of the Google doc.

**Comment by captaincobbs** [comment]
Going to change:
Mixed Anomalies renamed to Positive Anomalies, recolored appropriately
Positive Anomalies renamed to Enhanced Anomalies, recolored appropriately
Allow everyone to use anomaly enhancement, modify the effects of some ecologist laws to make them far more beneficial for ecologists.
Re-lore

**Comment by wakystuf** [comment]
That's all fine with me. What did you have in mind for the other Ecologist laws? I think most of them aside from the T0 are fine (and the T0 is great early, becoming useless only mid/late)

**Comment by captaincobbs** [comment]
Gave roughing it -10% Anomaly Enhancement/Reduction cost reduction per turn, up to 50% (15% up to 60% for republic). Doubled the cost of anomaly reductions and enhancements.

Moved all anomaly reductions to Simple Anomaly Reductions (T3)
Moved all anomaly enhancements to OCP Investigation (T4)

**Comment by captaincobbs** [comment]
It just needs to be lored, waiting on frost

**Comment by FluffiestPrincess** [comment]
Lore pass underway! #1393
